spectrum/src/main/java/com/thebluealliance/spectrum/SpectrumDialog.java

```diff
@@ -195,6 +197,17 @@ public Builder setSelectedColorRes(@ColorRes int selectedColorRes) {
         }
 
         /**
+         * Provides a resource that will be used to style to the dialog. See
+         * {@link AlertDialog#AlertDialog(Context, int)}.
+         *
+         * @return This {@link Builder} for method chaining
+         */
+        public Builder setThemeResId(int themeResId) {
```

Comment:
Do you want to mirror the android.view.AlertDialog.Builder constructor that takes a themeResId, rather than having a setter?